server/views/pages/offence/edit-offence.njk

```diff
@@ -59,6 +59,20 @@
                         }]
                     }
                 }].concat(summaryRows,[{
+                    key: {
+                        text: "Conviction date"
+                    }, value: {
+                        text: (offence.sentence.convictionDate | formatDate if offence.sentence and offence.sentence.convictionDate else 'N/A')
```

Comment:
you're already in an if statement here https://github.com/ministryofjustice/hmpps-remand-and-sentencing/pull/423/files#diff-4624e738415f013f715468bbeda39bca7ba921f3cd278bad12306572a0757897L46 so ` offence.sentence` condition is always true, can just drop this if statement down to `if offence.sentence.convictionDate`

Reply:
I have adjusted this code in the way you have described